src/App.js

```diff
@@ -0,0 +1,40 @@
+import './App.css';
+import {Home} from './home.js';
+import {Login} from './UserModule/login.js'
+import {Register} from './UserModule/register.js'
+import {HomeContainer} from './conatiners/HomeContainers.js'
+import {BrowserRouter ,Routes, Switch, Route} from 'react-router-dom';
```

Comment:
first import all external libraries and then add newline and import internal components/files.

Reply:
OKay